gwtquery-core/src/main/java/com/google/gwt/query/client/js/JsCache.java

```diff
@@ -135,7 +135,7 @@ public final native JsArrayMixed getArray(Object id) /*-{
   }
 
   public final native boolean isEmpty() /*-{
-    for (k in this) return false;
+    for (k in this) if (this.hasOwnProperty(k)) return false;
```

Comment:
There are many places in code doing this 'for ... if', it's a former tendency to save some characters in JSNI, I think it should be removed by optimizers, at least closure one (I have to check). Adding these curly brackets anyway.